go/engine/bug_3964_repairman_test.go

```diff
@@ -35,7 +35,9 @@ func (a *auditLog) Debug(format string, args ...interface{}) {
 	*a.lines = append(*a.lines, s)
 }
 func (a *auditLog) CDebugf(ctx context.Context, format string, args ...interface{}) {
-	a.l.CDebugf(ctx, format, args...)
+	s := fmt.Sprintf(format, args...)
+	a.l.CDebugf(ctx, s)
```

Comment:
It looks like you deleted the `format` arg here, when you meant to delete the `ctx` arg.

Reply:
I think it's correct as written?